inspire_matcher/validators.py

```diff
@@ -60,19 +59,15 @@ def authors_titles_validator(record, result):
 
     author_score = compute_author_match_score(record_authors, result_authors)
 
-    title_max_score = 0.0
     record_titles = get_value(record, 'titles.title', [])
     result_titles = get_value(result, '_source.titles.title', [])
 
-    for cartesian_pair in product(record_titles, result_titles):
-        record_title_tokens = get_tokenized_title(cartesian_pair[0])
-        result_title_tokens = get_tokenized_title(cartesian_pair[1])
-        current_title_jaccard = compute_jaccard_index(record_title_tokens, result_title_tokens)
+    title_score = max(
+        compute_title_score(record_title, result_title, threshold=0.5, math_threshold=0.3)
```

Comment:
Why math threshold is much lower than the normal one? Is mathml content so inconsistent that sometimes it's present and sometimes not?

Reply:
The problem is not that math is sometimes present and sometimes not, but that we don't have a good way of normalizing math across different representations. We get either of MathML/unicode/LaTeX, where arXiv will typically use the latter and publishers one of the two others. So we get things like
`Search for the decay $D^{0} \to \pi^{0} \nu \bar{\nu}$` and `Search for the decay D0→π0νν¯` which are morally the same but don't match as they don't reach the general threshold of 0.5 for the Jaccard index.